lib/engine/game/g_1880/game.rb

```diff
@@ -321,6 +322,41 @@ def event_signal_end_game!
           game_end_check
         end
 
+        def event_can_buy_trains!
+          @log << "-- Event: #{EVENTS_TEXT['can_buy_trains'][1]} --"
+          @can_buy_trains = true
+        end
+
+        def force_exchange_rocket
+          return unless @rocket
```

Comment:
In what case would @rocket not exist?

Reply:
rocket closed and was used before the forced use of the rocket

Reply:
fixed, i was using @rocket, instead of rocket. 
@rocket becomes null after it closes, i changed it to rocket.closed?